win32/tests/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ if(${run_unittests})
     endif()
 
     build_test_folder(gballoc_hl_passthrough_ut)
+    build_test_folder(job_object_helper_ut)

```

Comment:
>job_object_helper_ut [](http://example.com/codeflow?start=22&length=20)

I think we should add a task for an int test for this.
The int test could just try to allocate more than what the limit is for memory - to check that the job object is setup properly.

Reply:
We can do the same for CPU, we could spin n threads in a tight loop and check that CPU does not go over.
I highly suggest delegating this task to Parth so we do not put you with it on the critical path.